turbo.json

```diff
@@ -2,6 +2,7 @@
   "$schema": "https://turbo.build/schema.json",
   "pipeline": {
     "build:android": {
+      "env": ["ORG_GRADLE_PROJECT_newArchEnabled"],
```

Comment:
turbo build != turbo modules, but I did copy over this default env variable for new arch

Reply:
what is turbo build? 😮‍💨

Reply:
> Turborepo is a high-performance build system for monorepos, allowing you to manage multiple packages within a single repository efficiently. It optimizes build processes by caching and parallelizing tasks, resulting in faster development builds and improved performance.

I guess there's only so many adjectives in the english language lol. This was something that really boosted our CI performance especially iOS (but also caused those caching woes when we first added the GH actions)